alacritty_terminal/src/term/color.rs

```diff
@@ -85,7 +85,10 @@ impl<'de> Deserialize<'de> for Rgb {
         match value.deserialize_str(RgbVisitor) {
             Ok(rgb) => Ok(rgb),
             Err(err) => {
-                error!("Problem with config: {}; using color #000000", err);
+                error!(
+                    target: LOG_TARGET_CONFIG,
+                    "Problem with config: {}; using color #000000", err
```

Comment:
Either change the format of the color in the `err`, or use `0x000000` here.

Reply:
I wonder why we should change it? It's print that you have 2 options to set color and will say that it's using `#000000`.

Reply:
Oh, it depends on  #3383

Reply:
I could leave this comment on either PR, but this error message is starting to get rather long... I'd personally rather standardize on one, and then just leave the other in the config doc comments.

Reply:
Hm, let me try something.

Reply:
Hm, after trying to use the same color prefix format as the user specified it seems like it's pretty ugly and hacky, however it's possible to do something like that. For now I've decided to error that we have both formats.

>I'd personally rather standardize on one, and then just leave the other in the config doc comments.

Using only one format could be confusing and it could also force user to stick with the one format over another in the end.

Reply:
> Using only one format could be confusing and it could also force user to stick with the one format over another in the end.
 
I don't see any problem with this. Personally I'd be happy with everything being `#a0a0a0`, but I don't actually care nearly at all on this issue.

Reply:
The complexity of matching the users own format is definitely not worth it, there are much larger config issues deserving of complexity than this.

Reply:
See https://github.com/alacritty/alacritty/issues/3381#issuecomment-592158107.

That would solve all of this by just dropping all `0xRRGGBB` documentation.

Reply:
Synced.